### ISSUE TITLE
Update EGPF profile

### ISSUE THREAD

**Issue by 1adamf** [VATSIM-UK/uk-controller-pack#223]
**Is your feature request related to a problem? Please describe.**
Nope

**Describe the solution you'd like**
Update the Glasgow radar profile to a realistic profile

**Is your suggestion based on a real world display?**
Yes:
![Glasgow_Paul-43](https://user-images.githubusercontent.com/42348310/144707536-80dfdab3-74f6-43c5-b1c6-c148a5f5527b.jpg)
https://gyazo.com/fbcb37ddc771b2f186ba6fe4164f32a9

**Additional context**
Requires a sector file update for colour changes
Some of the River Clyde and the M74 are also missing from the sector file but will be a separate issue to the colour as I haven't added them yet.
Symbology for waypoints, VRPs and other stuff is also different to current sector file but a small enough difference that I ignored it when making the profile but someone else could add.

**Comment by 1adamf** [comment]
If it's not obvious as well I'm happy to do this if someone wants to assign it

**Comment by 1adamf** [comment]
I've had a play and apparently my euroscope knowledge has diminished because I can't work out how to replicate an old profile that I used ages ago which is now a bit outdated.

The colour changes I always used in the sector file were:

Land: 5329233
River: 11513680
coast: 11513680

Then by default:
Sids -> Glasgow SMAA is on
Geo -> Coastline (Medium detail) GB Main Coastline is on

I also used some different symbology settings in the screenshot which I can add but it's not massively different to the current default ones. The only difference that has quite a big effect is Symbology -> Sids -> Line -> Line weight = 2

I tried turning on rivers but the line for the Clyde actually extends much further in the sector file than in the image so I ended up turning it back off

Maybe someone else wants to have a shot at making the changes as it might be a while until I have time to properly try and work out why it's not working. (probably over the summer after uni has finished)
